pyproject.toml

```
@@ -0,0 +1,35 @@
+[build-system]
+requires = ["poetry-core>=1.0.0"]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry]
+name = "MIICE"
+version = "0.0.1"
+description = "Microscopy Image Illumination and Contrast Evaluation"
+readme = "README.md"
+license = "BSD-3-Clause"
+authors = ["Jenna Tomkinson <jenna.tomkinson@cuanschutz.edu>"]
+keywords = ["image-analysis", "cellpose", "microscopy", "illumination", "contrast"]
+packages = [{ include = "miice", from = "src" }]
+
+[tool.poetry.dependencies]
+python = ">=3.10,<4.0"
+matplotlib = "^3.10.0"
+cellpose = ">=4.0.6,<5.0"
+scikit-image = "^0.25.2"
+pandas = "^2.1.0"
```

Consider loosening these constraints:
```suggestion
matplotlib = ">=3.10.0"
cellpose = ">=4.0.6,<5.0"
scikit-image = ">=0.25.2"
pandas = ">=2.1.0"
```

---

Implemented!